### DISCUSSION TITLE
The requestPasswordReset method does not check whether the email is in the user table

### DISCUSSION THREAD

**Discussion by qxygene1** [pocketbase/pocketbase#6111] [Q&A]
The requestPasswordReset method does not check whether the email is in the user table before sending it, an error appears on the console (/_/#/logs?superuserRequests=1) **failed to fetch users record with email q@q.qq: sql: no rows in result set** but not on client side.

**Comment by ganigeorgiev** [comment]
It is working correctly and we always return a success response as a very basic user emails enumeration protection because the endpoint is public.

The error message is just for debugging purposes.
